whisper/src/lib.rs

```diff
@@ -580,15 +593,16 @@ fn adjust_instant_up(instant: u32, step: u32) -> u32 {
     (instant + step - 1) / step * step
 }
 
-fn adjust_interval(interval: Interval, step: u32) -> Result<Interval, String> {
+fn adjust_interval(interval: Interval, step: u32) -> Interval {
     let from_interval = adjust_instant(interval.from(), step);
     let until_interval = adjust_instant_up(interval.until(), step);
 
     if from_interval == until_interval {
         // Zero-length time range: always include the next point
-        Interval::new(from_interval, until_interval + step)
+        // should not panic
+        Interval::new(from_interval, until_interval + step).unwrap()
     } else {
-        Interval::new(from_interval, until_interval)
+        interval
```

Comment:
Looks broken. Merge this https://github.com/GiantPlantsSociety/graphite-rs/pull/191

Reply:
I see what I missed.